### DISCUSSION TITLE
Without using Argocd's 'Bundle Dex', Can I implement Github SSO in conjunction with ArgoCD using an already configured Standalone Dex Server?

### DISCUSSION THREAD

**Discussion by oliverpark999** [argoproj/argo-cd#19119] [Q&A]
I would like to implement Github SSO in conjunction with ArgoCD using the already configured Standalone Dex Server.
I can't find anything like this in the ArgoCD official documentation.
Is what I'm trying to implement a feasible scenario?
If possible, what should I do?

**Comment by ChristianCiach** [comment]
Your standalone Dex server should act like a regular IDP, so https://argo-cd.readthedocs.io/en/stable/operator-manual/user-management/#existing-oidc-provider should apply.
